docs/overview/learning-path-beginner.md

```diff
@@ -66,6 +66,14 @@ Congratulations on finishing the beginner's learning path for Office Add-ins! He
   - [Project quick start](../quickstarts/project-quickstart.md)
   - [Word tutorial](../tutorials/word-tutorial.md)
 
+- Scenarios and other code samples:
+
+  - [Outlook: Report spam or phishing emails](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/outlook-spam-reporting)
+  - [Outlook: Use SSO with event-based activation](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/auth/Outlook-Add-in-SSO-events)
```

Comment:
Since we're working on the NAA sample for event-based add-ins, maybe we could future proof this slightly by adding an Excel sample instead?
```suggestion
  - [Excel: Create a spreadsheet from your web page and embed your add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/excel-create-worksheet-from-web-site)
```